.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
     types: [created]
 
+env:
```

Comment:
These definitions are easy to follow if the reader knows what GITHUB_REF is.  I recommend a comment

```
# GITHUB_REF = refs/tags/STRING-MAJOR.MINOR
```

You also probably want to say that you are assuming the STRING does not contain a '-'.  So viewer and not cbmc-viewer.  Otherwise the definition of VERSION below fails.